src/main/scala/com/odsklm/salsabeach/GetOps.scala

```diff
@@ -151,7 +150,7 @@ object GetOps extends LazyLogging {
       maxTimestamp = maxTimestamp,
       versions = versions
     )
-    executeVersioned(key, get, columns)
+    executeVersioned(get, columns)
```

Comment:
Passing the key here was done on purpose because with a _get_ you can always be sure that the key you're requesting is also the key of the row itself (as opposed to a _scan_, where the keys can be different than the prefix). This reduces the need to deserialize the row key.

Of course, for consistency between scan and get, it might make more sense to actually deserialize the key instead of just passing it along.

Reply:
I also changed this because VersionedRow now needs a key that is a RowKeyDecoder, while the key that was passed here is of type RowQueryEncoder.